src/main/java/org/wildfly/security/x500/X500.java

```diff
@@ -137,4 +137,32 @@ private X500() {}
             return Arrays.copyOf(certificates, certificates.length, X509Certificate[].class);
         }
     }
+
+    /**
+     * Convert an X.500 attribute name (AVA keyword) to OID
+     * @param attribute X.500 attribute name
+     * @return corresponding OID or {@code null} if was not recognized
```

Comment:
1) I think others OIDs (or most of them) does not have assigned name to use in DN - or am I wrong?
2) In this method will be better to return null, to allow caller to detect if conversion was successful.
btw, sounds like instead of "oid"+"attribute-name" would be welcome one attribute "attribute" in subsystem... is it correct?